graphql-jpa-query-schema/src/test/java/com/introproventures/graphql/jpa/query/schema/model/calculated/CalculatedEntity.java

```diff
@@ -47,4 +48,15 @@ public String getHideFieldFunction() {
     }
 
     public void setSomething(int a){}
+
+    @GraphQLIgnore
+    public String getPropertyIgnoredOnGetter() {
+        return propertyIgnoredOnGetter;
+    }
+
+    @Transient
+    @GraphQLIgnore
```

Comment:
The purpose of these properties is more to check how `@GraphQLIgnore` annotation works rather than ignore transient fields by default. So there is one standard property and one annotated with `@Transient` just to check if everything works as it should. 😃